x/ccv/provider/keeper/relay.go

```diff
@@ -307,6 +356,13 @@ func (k Keeper) ValidateSlashPacket(ctx sdk.Context, chainID string,
 // a misbehaving validator according to infraction type.
 func (k Keeper) HandleSlashPacket(ctx sdk.Context, chainID string, data ccv.SlashPacketData) {
 
+	k.Logger(ctx).Debug("handling slash packet",
+		"chainID", chainID,
+		"consumer cons addr", sdk.ConsAddress(data.Validator.Address).String(),
```

Comment:
I believe this is the provider cons addr, see line 308. You could use the variable defined below this log
```suggestion
		"provider cons addr", sdk.ConsAddress(data.Validator.Address).String(),
```

Reply:
Nice catch